templates/logsearch-infrastructure-warden.yml

```diff
@@ -83,417 +83,417 @@ networks_default:
   subnets:
   - cloud_properties:
       name: random
-    range: 10.244.0.0/30
```

Comment:
You are right on that one, we need to get better at describing our commits for these bigger PRs